utils/telldus-core/Makefile

```diff
@@ -0,0 +1,67 @@
+# 
+# Copyright (C) 2010 Telldus Technologies AB
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=telldus-core
+PKG_VERSION:=2.1.2
+PKG_RELEASE:=1
+
+PKG_LICENSE:=LGPL-2.1
+PKG_BUILD_DEPENDS:=argp-standalone 
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download.telldus.com/TellStick/Software/telldus-core/
+PKG_HASH:=a20f6c74814afc23312d2c93ebbb37fdea9deaaee05ae7b6a6275e11e4662014
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/telldus-core
+	SECTION:=utils
+	CATEGORY:=Utilities
+	TITLE:=Telldus TellStick USB interface
+	URL:=http://www.telldus.com
+	MAINTAINER:=Peter Liedholm <PeterFromSwe884@gmail.com>
+	DEPENDS:=+confuse +libftdi +libstdcpp $(ICONV_DEPENDS)
+endef
+
+define Package/telldus-core/description
+	Telldus driver for USB-based 433 MHz RF transciever for home automation.
+	MUST be built with full language support, (don't know how to DEPEND that)
+endef
+
+CMAKE_OPTIONS+=\
+	-DBUILD_LIBTELLDUS-CORE=1 \
+	-DBUILD_TDTOOL=1 \
+	-DGENERATE_MAN=0 \
+	-DICONV_LIBRARY=-liconv
+
+define Package/telldus-core/conffiles
+/etc/tellstick.conf
+endef
+
+define Package/telldus-core/install
+	$(INSTALL_DIR)  $(1)/usr/bin
+	$(INSTALL_BIN)  $(PKG_INSTALL_DIR)/usr/bin/tdtool $(1)/usr/bin
+	$(INSTALL_DIR)  $(1)/usr/sbin
+	$(INSTALL_BIN)  $(PKG_INSTALL_DIR)/usr/sbin/telldusd $(1)/usr/sbin
+	$(INSTALL_BIN)  $(PKG_INSTALL_DIR)/usr/sbin/tdadmin $(1)/usr/sbin
+	$(INSTALL_DIR)  $(1)/usr/share/telldus-core/helpers
+	$(INSTALL_BIN)  $(PKG_INSTALL_DIR)/usr/share/telldus-core/helpers/udev.sh $(1)/usr/share/telldus-core/helpers
+	$(INSTALL_DIR)  $(1)/usr/include
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/include/telldus-core.h $(1)/usr/include
```

Comment:
Do you really want to install this header file on the device? I don't think it's needed unless you plan on compiling things on the device running openwrt..

Reply:
That's true, thanks for pointing it out!

Reply:
Header files belong in Build/InstallDev

Reply:
Header removed, not used on target at this scope.